code/modules/antagonists/zombie/zombie_spells.dm

```diff
@@ -42,7 +42,7 @@
 
 /obj/item/zombie_claw
 	name = "claws"
-	desc = "Claws extending from your rotting hands, perfect for ripping open brain_holders for brains."
+	desc = "Claws extending from your rotting hands, perfect for ripping skulls open for the brains inside."
```

Comment:
update to master please, you have a duplicated change here

Reply:
fggh made the change but forgot to swap branch -v-

Reply:
must've swapped branches weirdly actually? fixed though